handlers/extensions.go

```diff
@@ -37,7 +37,7 @@ func (h *Handler) ExtensionsEndpointHandler(w http.ResponseWriter, req *http.Req
 func (h *Handler) LoadExtensionFromPackage(w http.ResponseWriter, req *http.Request, provider models.Provider) error {
 	mx.Lock()
 	defer mx.Unlock()
-	packagePath := ""
+	packagePath := "/provider/navigator/meshmap/graphql/plugin.so" // default path gql plugin
```

Comment:
Please go back to the drawing board on this. We'll want to avoid setting a default to any given provider.